installing/installing_ibm_power/creating-ibm-power-vs-workspace.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: ASSEMBLY
+[id="creating-ibm-power-vs-workspace"]
+= Creating an {ibmpowerProductName} Virtual Server workspace
+include::_attributes/common-attributes.adoc[]
+:context: creating-ibm-power-vs-workspace
+
+:FeatureName: {ibmpowerProductName} Virtual Server using installer-provisioned infrastructure
+include::snippets/technology-preview.adoc[]
```

Comment:
The answer to this question is "No," you would not remove the following information: 

:_content-type: PROCEDURE
[id="creating-ibm-power-vs-workspace-procedure_{context}"]
== Steps to create an {ibmpowerProductName} Virtual Server workspace

It's not uncommon for the title of the Assembly to to match the title of the Procedure, for example: 

https://47558--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/graceful-cluster-shutdown.html

You should add content to describe the purpose of the Assembly. For example...

"This document describes how to create an IBM Power Virtual Server workspace." 

Like the above example, you might include prerequisites here if there are any.

Reply:
@stevsmit thanks. Incorporated the above suggestion.